src/sentry/features/__init__.py

```diff
@@ -57,6 +57,7 @@
 default_manager.add("organizations:api-keys", OrganizationFeature)
 default_manager.add("organizations:app-store-connect-multiple", OrganizationFeature, False)
 default_manager.add("organizations:boolean-search", OrganizationFeature)
+default_manager.add("organizations:breadcrumb-linked-event", OrganizationFeature, False)
```

Comment:
Do you want the third argument to be False or is that just a leftover from copy-paste?

Reply:
copy and paste... thanks